lib/irb.rb

```diff
@@ -207,7 +207,7 @@
 # in file <tt>~/.irb_history</tt>
 # (or, if a {configuration file}[rdoc-ref:IRB@Configuration+File]
 # is found, in file +.irb_history+
-# inin the same directory as that file).
+# in the same directory as that file).
```

Comment:
In addition to the typo, the entire sentence reads a bit weird 🤔 Would you mind improving it altogether?

Reply:
Pushed a larger change.